docs/_source/practical_guides/assign_records.md

```diff
@@ -15,162 +12,63 @@ You will need to decide the level of overlap before creating or pushing a datase
-
-```{note}
-This assumes that each annotator has a personal workspace attached to their user. If this is not the case, learn how to create a workspace and assign it to a user [here](/getting_started/installation/configurations/user_management.md#assign-a-user-to-a-workspace).
-```
+### Controlled overlap
 
-Here's how you can do this:
+We provide easy-to-use utility functions for distributing records among your team. Below, we outline a general approach for this task. For a more comprehensive example, please refer to our end-to-end tutorials.
 
-1. Get the list of users who will be annotating:
+1. Get the list of users who will be annotating and the records:
 
 ```python
 import argilla as rg
 
-rg.init(
-    api_url="...",
-    api_key="..."
-)
-
-# Get the list of users
 # Optional: filter users to get only those with annotator role
 users = [u for u in rg.User.list() if u.role == "annotator"]
-```
 
-```{note}
-If you are using a version earlier than 1.11.0 you will need to call the API directly to get the list of users. Note that, in that case, users will be returned as dictionaries and so `users.username` will be `users['username']` instead.
-```python
-# Make a request using your Argilla Client to get the list of users
-rg_client= rg.active_client().client
-auth_headers = {"X-Argilla-API-Key": rg_client.token}
-http=httpx.Client(base_url=rg_client.base_url, headers=auth_headers)
-users = http.get("/api/users").json()
-
-# Filter users to get only those with an annotator role
-users = [user for user in users if user["role"]=="annotator"]
+records = [...]
 ```
 
-2. Get a list of the records that each will annotate:
+2. Assign the annotations using the `assign_records` function:
 
 ```python
-import random
-from collections import defaultdict
-
-records = [...] # A Python list containing all the records in your dataset
-
-# Optional: shuffle the records to get a random assignment
-random.shuffle(records)
-
-# Build a dictionary where the key is the username and the value is the list of records assigned to them
-assignments = defaultdict(list)
-
-# Divide your records into chunks of the same length as the user list and make the assignments
-# You will need to follow the instructions to create and push a dataset for each of the key-value pairs in this dictionary
-n = len(users)
-chunked_records = [records[i:i + n] for i in range(0, len(records), n)]
-for chunk in chunked_records:
-    for idx, record in enumerate(chunk):
-        assignments[users[idx].username].append(record)
-```
-
-3. Loop through the dictionary of assignments to create one dataset per user:
-
-```{note}
-If you haven't done so already, decide on the settings of the project (the `fields`, `questions` and `guidelines`) as detailed in the [Create a Feedback Dataset guide](/practical_guides/create_dataset.md#feedback-dataset) and set those as variables.
+from argilla.client.feedback.utils import assign_records
+assignments = assign_records(users, records, 1, True)
```

Comment:
maybe you can use the explicit variable names `users=users`, `records=records` etc.